### ISSUE TITLE
obsolete comment in out_of_the_tarpit.hs 

### ISSUE THREAD

**Issue by agentm** [agentm/project-m36#78]
@3noch pointed out that the comment in the out_of_the_tarpit.hs example:

```
--custom types cannot yet be made due to hardcoding in the Atom Binary implementation
```
is obsoleted by the ConstructedAtom feature.

**Comment by agentm** [comment]
Actually, it looks like the example no longer builds. It should be automatically built as part of standard builds.

**Comment by agentm** [comment]
In addition, the documentation for creating a new Haskell data type is out-of-date, too. D'oh!

**Comment by 3noch** [comment]
Documentation has a way of doing that. ;)

**Comment by agentm** [comment]
Fixed in b5a81b3d60b5814f2da6b3a05fdd2efd84144740.
